modules/vstudio/vs2005_fsproj_user.lua

```diff
@@ -0,0 +1,91 @@
+--
+-- vs2005_csproj_user.lua
+-- Generate a Visual Studio 2005/2008 C# .user file.
+-- Copyright (c) 2009-2015 Jason Perkins and the Premake project
```

Comment:
First line should be `vs2005_fsproj_user.lua`, and probably full path. Second line should be `2005+ F# .user file`. Third line should be `2017-2017`.